eclass/kernel-build.eclass

```diff
@@ -41,6 +41,50 @@ BDEPEND="
 	app-alternatives/yacc
 "
 
+IUSE="+strip"
+
+# @ECLASS_VARIABLE: ALLOW_MODULES_SIGN
```

Comment:
Perhaps prefix it with `KERNEL_`, and use correct gramar, i.e. `KERNEL_ALLOW_MODULE_SIG` (`MODULE_SIG` to match kernel config).

Reply:
`MODULES_SIGN` on the other hand matches the use flag.

Maybe we also need to revisit the word 'allow' here, because it is not that it is not allowed to sign the modules if it is not set (it will still work via the user config route).

Reply:
What about `KERNEL_MODULES_SIGN_OPT`? This reflects better that we are adding an option and not allowing something that would otherwise not be allowed. 

I've thought about `KERNEL_MODULE_SIG_OPT` but I would prefer to keep consistent with the use flag rather then consistent with the kernel config option. Unfortunately we can't do both.